linter/whitelists/de.txt

```diff
@@ -296,6 +296,8 @@ Paginierungen
 Panel
 Panelen
 Panels
+Parallax
+parallax
```

Comment:
I would have added these words to the English whitelist, but, well, okay.

Reply:
But they are used in the german translation as well?

Reply:
>  would have added these words to the English whitelist, but, well, okay.

They don't need to be, unless the spell checker flagged them?

Reply:
Btw. you only use `Parallax` in your German meta data, not `parallax`. The latter should be removed.